packages/containerd-2.0/1002-Upgrade-systemd-config-for-packaging.patch

```diff
@@ -0,0 +1,44 @@
+From b2f0276d99195052870e0d69b0da75f229b82c7b Mon Sep 17 00:00:00 2001
```

Comment:
You don't need this patch. Instead make sure that our `containerd.service` includes the same configurations suggested in the patch.